test/bigtest/tests/loan-history/loan-history-form-test.js

```diff
@@ -162,4 +162,129 @@ describe('Loan History Form', () => {
       });
     });
   });
+
+  describe('when form is loaded and checkbox "Treat closed loans ..." is checked', () => {
```

Comment:
I would not recommend to hardcode the names like "Treat closed loans ..." in the description as they can be later be changed without updating test description. Please just describe the name like `checkbox for ...`